spyder/plugins/editor/widgets/codeeditor.py

```diff
@@ -2813,6 +2813,9 @@ def setup_context_menu(self):
         zoom_reset_action = create_action(
             self, _("Zoom reset"), shortcut=QKeySequence("Ctrl+0"),
             triggered=self.zoom_reset.emit)
+        insert_filepath = create_action(
+            self, _("Insert file path"), shortcut=QKeySequence(""),
+            triggered=self.insert_filepath)

```

Comment:
Please also add this action to our *Source* menu.

Reply:
@ccordoba12 Oh nice, I'll do it right away! Thanks

Reply:
Shoud I remove it from the *right-click menu*? Or just add to the *source* menu?

Reply:
Please add it to the source menu and leave it in the right-click one.

Reply:
Ok!